sympy/series/tests/test_limits.py

```diff
@@ -456,6 +456,11 @@ def test_issue_8730():
     assert limit(subfactorial(x), x, oo) == oo
 
 
+def test_issue_10801():
+    # make sure limits work with binomial
+    assert limit(16**k/(k*binomial(2*k,k)**2), k, oo) == pi
```

Comment:
Actually [here](https://github.com/sympy/sympy/tree/master/sympy/series/gruntz.py#L411) the expression is converted into tractable form. In the case ``_eval_rewrite_as_tractable = _eval_rewrite_as_gamma``, ``binomial`` is just converted into ``gamma`` form. The current implementation wants  ``gamma`` to also be converted into tractable form.